components/color/color.css

```diff
@@ -0,0 +1,61 @@
+@import "../../assets/common-css/navbar.css";
+@import "../../document-css/doc.css";
+@import "../../assets/common-css/sidebar.css";
+@import "../../assets/common-css/sidebar.css";
+
+.color-intro, .color-head{
+    color: var(--font-color);
+    font-size: 1.7rem;
+}
+.color-head{
+    margin-bottom: 1.3rem;
+}
+.color-intro{
+    margin: -5px;
+}
+.color-content{
+    padding: 20px;
```

Comment:
Like mentioned above, prefer to use rems, and define size variables in another css file and use those variables here